package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@roadiehq/backstage-plugin-jira",
-  "version": "1.0.1",
+  "version": "1.0.2",
```

Comment:
Out of curiosity, do we need to bump the version to get the docs to npm ? or does it read from github? While it's no big deal normally a docs change wouldn't  need a version bump; Feel free to merge this anyway.